apis/telemetry/v1alpha1/metricpipeline_types.go

```diff
@@ -89,6 +89,8 @@ type MetricPipelineIstioInput struct {
 type MetricPipelineOutput struct {
 	// Defines an output using the OpenTelemetry protocol.
 	Otlp *OtlpOutput `json:"otlp"`
+	// Defines whether this metricpipeline should convert monotonic, cumulative sum and histogram metrics to monotonic, delta metrics.
```

Comment:
[Camel case for Kubernetes resources](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md).
```suggestion
	// Defines whether this MetricPipeline should convert monotonic, cumulative sum and histogram metrics to monotonic, delta metrics.
```